source/_integrations/go2rtc.markdown

```diff
@@ -40,6 +40,10 @@ url:
   type: string
 {% endconfiguration %}
 
+If you using the go2rtc server managed by Home Assistant, please be aware that all ports are prefixed by `1` to avoid port conflicts:
```

Comment:
```suggestion
If you using the go2rtc server managed by Home Assistant, please be aware that all ports are prefixed by `1` compared to the default port settings to avoid port conflicts:
```